sct-commons/src/main/java/org/lfenergy/compas/sct/commons/ExtRefEditorService.java

```diff
@@ -46,6 +48,7 @@ public class ExtRefEditorService implements ExtRefEditor {
 
     private final IedService iedService;
     private final LdeviceService ldeviceService;
+    private final LnEditor lnEditor;
```

Comment:
why not LnService ?

Reply:
It is always a good practice to code with interfaces. Dependency Inversion Principle of SOLI**D** 

Related task https://github.com/com-pas/compas-sct/issues/110